tests/units/test_var.py

```diff
@@ -993,21 +992,14 @@ def test_string_operations():
 def test_all_number_operations():
```

Comment:
leave this test the way it was, if it's failing, fine, i'll look at it myself, but don't change this test at all.